addon-test-support/ember-mocha/setup-application-test.js

```diff
@@ -4,7 +4,8 @@ import {
 } from '@ember/test-helpers';
 import setupTest from './setup-test';
 
-export default function setupApplicationTest(options) {
+export default function setupApplicationTest(_options) {
+  let options = _options === undefined ? { waitForSettled: true } : assign({ waitForSettled: true }, _options);
```

Comment:
This change is probably unnecessary given the call to `setupTest` that follows. I can take it out if need be.